tests/test.yml

```diff
@@ -40,7 +40,6 @@
              address: "{{ internal_ip }}"
              netmask: 255.255.255.0
        - bridges: [ "br-example" ]
-       - fqdn: "localhost"
```

Comment:
should this be replaced with something?

Reply:
No, IMHO. It should really on the inventory to work. If I should not remove that variable the test might not be correctly tested.